### PR TITLE
sideboard's first commit

### PR DESCRIPTION
**Pull request by sideb0ard**
Following GitHub codelab

b/12345

### CODE REVIEW

File: sandbox/sideboard.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hi from sideboard in sandbox
```
Comment (kaidokert) [inline sandbox/sideboard.txt:1]: Command and caps missing !

Comment (kaidokert) [reply sandbox/sideboard.txt:1]: Sorry i meant a missing comma !

Comment (sideb0ard) [reply sandbox/sideboard.txt:1]: done!

**Review by kaidokert — APPROVED** [review]
Looks fancy to me

**Comment by jellefoks** [comment]
Left a Comment right here for your enjoyment.

**Pull request merged**